src/Psalm/Internal/Analyzer/Statements/Expression/Fetch/VariableFetchAnalyzer.php

```diff
@@ -531,26 +540,231 @@ public static function getGlobalType(string $var_id): Union
         }
 
         if ($var_id === '$argv') {
+            // only in CLI, null otherwise
```

Comment:
You're right, but if we keep this as-is it will be a pain for users. Can you set `Union::ignore_nullable_issues` to true for both variables?

It will allow for nullable checks while still being lax enough to avoid reporting annoying `this may be null` errors

Reply:
There's too much code where we can guarantee we're in CLI (I'm thinking about Commands in Symfony for example that are pieces of code only made to be ran on CLI). Adding CLI checks on that kind of code would be redundantly painful IMO.

I'm fine adding a config about this though in order to enable a stricter check for those